server/api/healthz.get.ts

```diff
@@ -14,6 +14,6 @@ export default defineEventHandler((): ResponseHealthcheck => {
     status: 'healthy',
     time: new Date(),
     startupTime,
-    nuxtAppVersion: process.env.NUXT_APP_VERSION || 'unknown',
+    nuxtAppVersion: process.env.npm_package_version || 'unknown',
```

Comment:
let's still allow to set this via env var first, so:
```suggestion
    nuxtAppVersion: process.env.NUXT_APP_VERSION || process.env.npm_package_version || 'unknown',
```

because, as pointed out yesterday: the npm_package_version can onl;y be set "at compile time" of the final docker image. env vars can be set at runtime!